### ISSUE TITLE
UI shift when archiving/unarchiving an experiment

### ISSUE THREAD

**Issue by pdehaan** [mozilla/experimenter#6244]
Wow! Since when can I upload .mp4's directly to GitHub without converting to .gif? Super rad!

Toggling the Archive/Unarchive Experiment button causes some slight UI shifting when you toggle the button. As close as I can tell, this is limited to the 2-3 breakpoints where the "Archive Experiment" text wraps onto two lines. It's less noticeable if the text is on a single line.

https://user-images.githubusercontent.com/557895/129820354-474054ef-6e97-490f-a280-a9674e75fc60.mp4

**Comment by pdehaan** [comment]
Looking at the video above again, I'm referring to the _horizontal_ text shifting. The _vertical_ shifting is expected because of the toggling of "Missing details in: Audience" text which is modified depending on the current state.

**Comment by jaredlockhart** [comment]
Yeah I noticed it too.  I'll leave it as a good first bug if a contributor wants to poke around at it 👍

**Comment by pdehaan** [comment]
Interestingly, I think the default _centered_ text is the bug. It seems like when an experiment is archived and it temporarily left aligns the text, that looks "right" since the other buttons+labels are all consistently left aligned above.
